src/reifyhealth/specmonstah/core.cljc

```diff
@@ -518,6 +519,11 @@
                  (:types db))
          (add-ref-ents))))
 
+(defn build-ent-db
+  [db query]
+  (println "This was deprecated in 2.0.0-alpha-2 and will be removed in 2.0.0")
```

Comment:
May want to consider adding what is deprecated so it stands out in logs, eg:
```
`r.s.core/build-ent-db` was deprecated in 2.0.0-alpha-2 in favor of `r.s.core/add-ents` and will be removed in 2.0.0"
```

Additionally, may be worth adding a `^{:deprecated 2.0.0-alpha-2}` meta tag for auto-docs, etc.

Reply:
I think it would also be preferable to have this go to `*err*` rather than `*out*`.